### PR TITLE
No jsoncpp

### PR DESCRIPTION
**Pull request by jschueller**
Avoids a circular dependency when building jsoncpp, cc @jakirkham

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @jschueller. This should be easier.

File: recipe/build.sh [hunk 1]
```diff
@@ -22,9 +13,10 @@ fi
              --system-libs \
              --no-qt-gui \
              --no-system-libarchive \
+             --no-system-jsoncpp \
```
Comment (isuruf) [inline recipe/build.sh]: While we are changing this, we can add `--parallel=${CPU_COUNT}`

Comment (jschueller) [inline recipe/build.sh]: I already added make -jCPU_COUNT

Comment (isuruf) [inline recipe/build.sh]: Yes, but that's for the build step after bootstrap phase is done. We can do the bootstrapping parallely as well.

Comment (jschueller) [inline recipe/build.sh]: ok

**Comment by jschueller** [comment]
good to go ?

**Comment by jakirkham** [comment]
Seems fine. Though am away from keyboard.

**Comment by jschueller** [comment]
Could someone with write access merge this ?

**Pull request merged**